modules/UI/videolayout/RemoteVideo.js

```diff
@@ -164,7 +163,9 @@ RemoteVideo.prototype._generatePopupContent = function () {
     let remoteControlState = null;
     let onRemoteControlToggle;
 
-    if (this._supportsRemoteControl) {
+    if (this._supportsRemoteControl
+        && ((!APP.remoteControl.active && !this._isRemoteControlSessionActive)
+            || APP.remoteControl.controller.activeParticipant === this.id)) {
```

Comment:
It is necessary. `APP.remoteControl.controller.activeParticipant` is storing the id of the user who was requested by the local user for remote control session or has an ongoing remote control session with the local user. `this._isRemoteControlSessionActive` is `true` if the user related to a RemoteVideo has active remote control session. For example:
User A, B, C are in the conference. If A and B have active remote control session than the values of the variables for C are going to be `APP.remoteControl.controller.activeParticipant = undefined` and `this._isRemoteControlSessionActive = true` for the RemoteVideos of A and B.

When we reactify the RemoteVideo and the RemoteControl probably we'll want to put this in redux and maybe we'll need to change a little bit the format of the data but I don't think we need to change it right now and I'm not sure that the format will be very different.